src/libtrx/include/libtrx/game/objects/traps/movable_block.h

```diff
@@ -4,5 +4,23 @@
 
 void MovableBlock_Initialise(int16_t item_num);
 void MovableBlock_UpdateRotation(ITEM *item, int16_t rot_y);
-void MovableBlock_SetupFloor(void);
-void MovableBlock_HandleFlipMap(ROOM_FLIP_STATUS flip_status);
+void MovableBlock_UpdateBox(const ITEM *item, bool blocked);
+void MovableBlock_SetPushPull(ITEM *item, bool enable);
+bool MovableBlock_IsPushPull(const ITEM *item);
+void MovableBlock_SetGravityFrames(ITEM *item, uint8_t frames);
+uint16_t MovableBlock_GetGravityFrames(const ITEM *item);
+void MovableBlock_SetPos(ITEM *item);
+XYZ_32 MovableBlock_GetPos(const ITEM *item);
```

Comment:
I'd name these functions, `SetPos` and `GetPos`, something that stands out more, as currently it's unclear how `MOVABLE_BLOCK_INFO.pos` is different from `ITEM.pos` and how it should be used.

Reply:
For the movable blocks and sliding pillars, the `item->data`'s positions is sort of a last known, good, old position. It's needed for both to reposition their walkable without allocating more memory. I wasn't 100% sure what to call it either. I'll think on it.

Reply:
I've decided to go with `linked_sector_pos`. I think that's the best description. It's basically the last good, sector location.